karapace/schema_registry_apis.py

```diff
@@ -736,10 +780,29 @@ async def _subject_version_delete_local(self, content_type: str, subject: str, v
                 status=HTTPStatus.NOT_FOUND,
             )
 
+        referenced_by = self.ksr.referenced_by.get(reference_key(subject, version), None)
+        if referenced_by and len(referenced_by) > 0:
+            self.r(
+                body={
+                    "error_code": SchemaErrorCodes.SCHEMAVERSION_HAS_REFERENCES.value,
```

Comment:
I cross checked this error code and correct would be: https://github.com/confluentinc/schema-registry/blob/master/core/src/main/java/io/confluent/kafka/schemaregistry/rest/exceptions/Errors.java#L63